data/data/manifests/bootkube/kube-system-secret-etcd-client-ca-deprecated.yaml.template

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: etcd-client-ca-deprecated
```

Comment:
Why are we opening with a deprecated object?

Reply:
> Why are we opening with a deprecated object?

no sure what you mean by that.

This mimics the deprecation comment on https://github.com/openshift/installer/blob/c52894b2c2064e5cb8a6d736ccdbe29279d18ceb/pkg/asset/tls/etcd.go#L11
anf https://github.com/openshift/installer/blob/c52894b2c2064e5cb8a6d736ccdbe29279d18ceb/pkg/asset/manifests/operators.go#L168